ex5_v0.py

```diff
@@ -0,0 +1,36 @@
+
+def longueur_ligne():
+	nb_pts = len(ligne)
+	longueur = 0
+	while nb_pts > 1 :
```

Comment:
Why do you iterate from the end of the line ? It works but it is unusual.
You should use a "for x, y in line:" here, to be more explicit